source/API_Reference/Web_API_v3/cancel_scheduled_send.apiblueprint

```diff
@@ -0,0 +1,139 @@
+---
+layout: page
+weight: 0
+title: Cancel Scheduled Sends
+navigation:
+  show: true
+---
+
+FORMAT: 1A
+
+# Group Batch IDs
+
+If you set the [SMTPAPI header batch_id]({{root_url}}/API_Reference/SMTP_API/scheduling_parameters.html),
+it allows you to then cancel a scheduled send based on that batch_id by calling the [Cancel Scheduled Send endpoint]({{root_url}}/API_Reference/Web_API_v3/cancel_scheduled_send.html#Cancel-Scheduled-Sends)
+
+## Batch ID Generation [/mail/batch]
+Generate a new Batch ID to associate with scheduled sends
+
+### Generate Batch ID [POST]
+
++ Request
+
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "batch_id": "YOUR_BATCH_ID",
+                "expiration": 1442945818
+            }
+
+## Validate Batch ID [/mail/batch/]
+
+### Validate Batch ID [GET]
+
+Validate whether or not a batch id is valid
+
++ Request
+
++ Response 200 (application/json)
+
+    + Body
+
+             {
+                "batch_id": "YOUR_BATCH_ID",
```

Comment:
Batch id is included in the url, not the body now
for example:
/mail/batch/MjNmOWEzYjItNjYxMi0xMWU1LTllMzYtMDgwMDI3OGJkMmY2LTQyOTI4YWUyNw